### ISSUE TITLE
DevTools failed to load SourceMap

### ISSUE THREAD

**Issue by RinVindor** [HadaIonut/Equipement-Paper-Doll#15]
I'm getting a warning in console figured I'd make note of it here: I'm running the latest version  1.0.5 and I'm on Foundry 0.8.8 and D&D 5E 1.3.6

DevTools failed to load SourceMap: Could not load content for http://localhost:30000/modules/Equipment-Paper-Doll/scripts/lib/popperJs/popper.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by HadaIonut** [comment]
Yes, thank you, but I am aware of it and not exactly sure how to make it go away.

The warning comes from a library for tooltips and to my knowledge does not affect anything. But anyway thx for pointing it out :)
